tests/aggregation/tests.py

```diff
@@ -2158,6 +2160,15 @@ def test_having_with_no_group_by(self):
         )
         self.assertEqual(list(author_qs), [337])
 
+    def test_JSONArrayAgg(self):
+        try:
+            vals = Author.objects.filter(age__gt=29).aggregate(
+                age_array=JSONArrayAgg("age")
```

Comment:
It'd be good to have tests for the aggregation of non-JSON compatible types such a `DateTimeField` and such.